torchreid/models/mobilenetv3.py

```diff
@@ -254,7 +306,7 @@ def init_pretrained_weights(model, key=''):
     def _get_torch_home():
         ENV_TORCH_HOME = 'TORCH_HOME'
         ENV_XDG_CACHE_HOME = 'XDG_CACHE_HOME'
-        DEFAULT_CACHE_DIR = '~/.cache'
+        DEFAULT_CACHE_DIR = './.cache'
```

Comment:
For some reason on the cluster, my home directory leaked through when downloading model weights... And I decided to change the directory

Reply:
Now, there is no problem anymore, but I agree that it would be harder to locate all the cache folders. Maybe just to write something like "/home/.pytorch/checkpoints"?

Reply:
On the cluster you can workaround that issue by exporting TORCH_HOME=./

Reply:
ok. I returned this back